packages/ui/src/components/va-skip-link/VaSkipLink.stories.ts

```diff
@@ -0,0 +1,41 @@
+import { VaRadio } from '../va-radio'
+import { VaSkipLink } from './'
+
+export default {
+  title: 'VaSkipLink',
+  component: VaSkipLink,
+}
+
+export const Default = () => ({
+  components: { VaSkipLink },
+  template: `
+    <va-skip-link target="#target">
+      [skip link]
+    </va-skip-link>
+    <div :style="{ height: '100dvh' }">
+      [not target]
+    </div>
+    <div 
+      id="target" 
+      :style="{ height: '100dvh' }"
+    >
+      [target]
+    </div>
+  `,
+})
+
+const OPTIONS = ['top-left', 'top-right', 'bottom-left', 'bottom-right']
+
+export const Position = () => ({
```

Comment:
Can't we add an easy interaction test here?

Maybe even only for one case.

The problem is that we have solid visual tests, but we won't click around for every story. But we need to somehow ensure core functionality stands.